server/main.tf

```
@@ -0,0 +1,27 @@
+resource "aws_instance" "basion_instance" {
+  ami           = var.ami
+  instance_type = var.instance_type
+  associate_public_ip_address = true
+  key_name = var.key_name_basion
+  subnet_id = aws_subnet.public_subnets[1].id
```

```suggestion
  subnet_id = element(var.private_subnet_ids, 1)
```
lấy 1 id ở trong list `private_subnet_ids`